Inc/Audio.h

```diff
@@ -354,6 +361,10 @@ namespace DirectX
 
         void __cdecl UnregisterInstance(_In_ IVoiceNotify* instance);
 
+        HANDLE __cdecl GetAsyncHandle() const noexcept;
+
+        bool __cdecl GetPrivateData(unsigned int index, _Out_writes_bytes_(datasize) void* data, size_t datasize);
```

Comment:
This was needed to break the abstraction a little bit for SoundStreamInstance to get some additional data without having to expose the whole pimpl.